Watchman.DomainModel/Users/MuteEvent.cs

```diff
@@ -21,12 +24,13 @@ public bool Unmuted
 
         private bool _unmuted;
 
-        public MuteEvent(ulong userId, TimeRange timeRange, string reason, ulong serverId)
+        public MuteEvent(ulong userId, TimeRange timeRange, string reason, ulong serverId, ulong channelId)
         {
             this.UserId = userId;
             this.TimeRange = timeRange;
             this.Reason = reason;
             this.ServerId = serverId;
+            this.ChannelWhereGivenId = channelId;
```

Comment:
ja to rozumiem "kanał na którym dostał ID"
a nie jako "id kanału na którym został zmutowany"

Reply:
plus myśl o kompatybilności wstecznej

Reply:
no w sumie
tylko jak nazwać inaczej?

Reply:
może `MutedOnChannelId`

Reply:
albo In... nie jestem ekspertem od angielskiego, ale chyba `on`

Reply:
tylko że wiesz, żeby była kompatybilność wsteczna

Reply:
może lepiej będzie nie zmieniać tego, żeby nie robić problemów

Reply:
wcześniej wgl nie było
ale teraz jest potrzebny channelId
kompatybilność wstecznej nie będzie (jeśli mam zmienić Unmuted na IsUnmuted), ale i tak na razie nie jest to dużym problemem bo wszystkie muteEvents są unmuted

Reply:
myśle że `MutedOnChannelId` będzie ok